modules/dnn/perf/perf_net.cpp

```diff
@@ -71,6 +76,14 @@ class DNNTestNetwork : public ::perf::TestBaseWithParam< tuple<Backend, Target>
 
         SANITY_CHECK_NOTHING();
     }
+
+    void processNet(std::string weights, std::string proto, std::string halide_scheduler,
+                    Mat input, const std::string& outputLayer = "")
+    {
+        randu(input, 0.0f, 1.0f);
```

Comment:
I think if user passed specific input - it's expected to be initialized. Might be important for models which performance depend on data (like two-stage detectors).

Reply:
Actually, I have already changed it. If user just need random number, they can pass a un-initialized Mat as input. If they need specific input, they can pass the Mat directly to the new API with `{{input, ""}}`